recipes/gettext/all/conanfile.py

```diff
@@ -107,12 +112,8 @@ def package(self):
                     env_build = self._configure_autotools()
                     env_build.install()
         tools.rmdir(os.path.join(self.package_folder, 'share'))
-        tools.rmdir(os.path.join(self.package_folder, 'lib'))
-        tools.rmdir(os.path.join(self.package_folder, 'include'))
-
-    def package_id(self):
-        self.info.include_build_settings()
-        del self.info.settings.compiler
+        os.rename(os.path.join(self.package_folder, "include", "libgnuintl.h"),
+                  os.path.join(self.package_folder, "include", "libintl.h"))
```

Comment:
This surprise me a lot... 🤔

Reply:
this comes from https://github.com/conan-io/conan-center-index/blob/master/recipes/libgettext/all/conanfile.py#L118

Reply:
Ok, but it keeps surprising me 😆